ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -665,156 +595,184 @@ export default class AdvancedTab extends PureComponent {
           </div>
         </div>
         <div className="settings-page__content-item">
           <div className="settings-page__content-item-col">
-            <TextField
-              type="text"
-              value={this.state.ipfsGateway}
-              onChange={(e) => this.handleIpfsGatewayChange(e.target.value)}
-              error={ipfsGatewayError}
-              fullWidth
-              margin="dense"
+            <ToggleButton
+              value={dismissSeedBackUpReminder}
+              onToggle={(value) => setDismissSeedBackUpReminder(!value)}
+              offLabel={t('off')}
+              onLabel={t('on')}
             />
-            <Button
-              type="primary"
-              className="settings-tab__rpc-save-button"
-              disabled={Boolean(ipfsGatewayError)}
-              onClick={() => {
-                this.handleIpfsGatewaySave();
-              }}
-            >
-              {t('save')}
-            </Button>
           </div>
         </div>
       </div>
     );
   }
 
-  renderDismissSeedBackupReminderControl() {
+  handleLockChange(time) {
     const { t } = this.context;
-    const { dismissSeedBackUpReminder, setDismissSeedBackUpReminder } =
-      this.props;
+    const autoLockTimeLimit = Math.max(Number(time), 0);
+
+    this.setState(() => {
+      let lockTimeError = '';
+
+      if (autoLockTimeLimit > 10080) {
+        lockTimeError = t('lockTimeTooGreat');
+      }
+
+      return {
+        autoLockTimeLimit,
+        lockTimeError,
+      };
+    });
+  }
 
+  renderUserDataBackup() {
+    const { t } = this.context;
     return (
       <div
-        ref={this.settingsRefs[11]}
+        ref={this.settingsRefs[12]}
         className="settings-page__content-row"
-        data-testid="advanced-setting-dismiss-reminder"
+        data-testid="advanced-setting-data-backup"
       >
         <div className="settings-page__content-item">
-          <span>{t('dismissReminderField')}</span>
-          <div className="settings-page__content-description">
-            {t('dismissReminderDescriptionField')}
-          </div>
+          <span>{t('backupUserData')}</span>
+          <span className="settings-page__content-description">
+            {t('backupUserDataDescription')}
+          </span>
         </div>
         <div className="settings-page__content-item">
           <div className="settings-page__content-item-col">
-            <ToggleButton
-              value={dismissSeedBackUpReminder}
-              onToggle={(value) => setDismissSeedBackUpReminder(!value)}
-              offLabel={t('off')}
-              onLabel={t('on')}
-            />
+            <Button
+              data-testid="backup-button"
+              type="secondary"
+              large
+              onClick={() => this.backupUserData()}
+            >
+              {t('backup')}
+            </Button>
           </div>
         </div>
       </div>
     );
   }
 
-  renderTokenDetectionToggle() {
```

Comment:
The only change logic-wise in this file, is that this method (and attendant variables) was removed and moved to security tab component.